docs/examples/physics_using_pybullet/viz_domino.py

```diff
@@ -0,0 +1,226 @@
+    baseOrientation=base_orientation)
+
+p.changeDynamics(base, -1, lateralFriction=1, restitution=0.5)
+
+###############################################################################
+# We define some global parameters of the Dominoes so that its easier for
+# us to tweak the simulation.
+
+domino_mass = 0.5
+domino_size = np.array([0.1, 1, 2])
+
+number_of_dominoes = 10
+domino_centers = np.zeros((number_of_dominoes, 3))
+
+# Keeping all the dominos Parallel
+domino_directions = np.zeros((number_of_dominoes, 3))
+domino_directions[:] = np.array([1.57, 0, 0])
+
+domino_orns = np.zeros((number_of_dominoes, 4))
+
+domino_sizes = np.zeros((number_of_dominoes, 3))
+domino_sizes[:] = domino_size
+
+domino_colors = np.random.rand(number_of_dominoes, 3)
+
+domino_coll = p.createCollisionShape(p.GEOM_BOX,
+                                     halfExtents=domino_size / 2)
+
+# We use this array to store the reference of domino objects in pybullet world.
+dominos = np.zeros(number_of_dominoes, dtype=np.int8)
+
+centers_list = np.zeros((number_of_dominoes, 3))
+
+# Adding the dominoes
+for i in range(number_of_dominoes):
+    center_pos = np.array([(i*0.99)-5.5, 0.4, 1])
+    domino_centers[i] = center_pos
+    domino_orns[i] = np.array([0, 0, 0, 1])
+    dominos[i] = p.createMultiBody(baseMass=domino_mass,
+                                   baseCollisionShapeIndex=domino_coll,
+                                   basePosition=center_pos,
+                                   baseOrientation=domino_orns[i])
+    p.changeDynamics(dominos[i], -1, lateralFriction=0.2, restitution=0.1)
+
+
+domino_actor = actor.box(centers=domino_centers,
+                         directions=domino_directions,
+                         scales=domino_sizes,
+                         colors=domino_colors)
+
+###############################################################################
+# Now, we define a scene and add actors to it.
+scene = window.Scene()
+scene.add(actor.axes())
+scene.add(base_actor)
+scene.add(domino_actor)
+
+# Create show manager.
+showm = window.ShowManager(scene,
+                           size=(900, 768), reset_camera=False,
+                           order_transparent=True)
+
+showm.initialize()
+
+# Counter iterator for tracking simulation steps.
+counter = itertools.count()
+
+# Variable for tracking applied force.
+apply_force = True
+###############################################################################
+# Now, we define methods to sync objects between fury and Pybullet.
+
+# Get the position of base and set it.
+base_pos, _ = p.getBasePositionAndOrientation(base)
+base_actor.SetPosition(*base_pos)
+
+
+# Calculate the vertices of the dominos.
+vertices = utils.vertices_from_actor(domino_actor)
+num_vertices = vertices.shape[0]
+num_objects = domino_centers.shape[0]
+sec = np.int(num_vertices / num_objects)
+
+###############################################################################
+# ==============
```

Comment:
`===` should go until the end of `Dominoes`, otherwise there is a error on doc generation